lib/vm.js

```diff
@@ -73,18 +75,57 @@ Script.prototype.runInContext = function(contextifiedSandbox, options) {
 };
 
 Script.prototype.runInNewContext = function(sandbox, options) {
-  var context = createContext(sandbox);
+  const contextOptions = options ? {
```

Comment:
Not a super big deal, but should this code section be extracted into a separate function, since it's also used below?